ADAL/src/request/ADWebAuthResponse.m

```diff
@@ -394,7 +394,7 @@ - (void)handleJSONError:(NSError*)jsonError
         }
         
         AD_LOG_ERROR(_request.correlationId, @"JSON deserialization error:");
-        AD_LOG_ERROR_PII(_request.correlationId, @"%@ - %@", jsonError.description, bodyStr);
+        AD_LOG_ERROR_PII(_request.correlationId, @"JSON deserialization error: %@ - %@", jsonError.description, bodyStr);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 121 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)